src/plugins/expressions/common/expression_types/kibana_datatable.ts

```diff
@@ -23,9 +23,16 @@ import { Datatable, PointSeries } from '.';
 
 const name = 'kibana_datatable';
 
+export interface KibanaDatatableColumnMeta {
+  type: string;
+  indexPatternId: string;
+  params: Record<string, any>;
```

Comment:
```suggestion
  aggConfig: Record<string, unknown>;
```

Having a generic object called `params` is not helpful for our use in client apps- will we need to assume that `params` is always the same data?

I suggest renaming to `aggConfigs` because that's what it appears to be in this change.

Reply:
its aggConfigParams